glib/src/value.rs

```diff
@@ -205,6 +205,29 @@ pub unsafe trait FromValue<'a>: Sized {
     unsafe fn from_value(value: &'a Value) -> Self;
 }
 
+/// Similar to `FromValue` but works for owned `Value.
+///
+/// This example cannot compile because &str is a temporary value.
+/// ```compile_fail
+/// let x = "gtk-rs".to_value();
+///
+/// let y = x.get_owned::<&str>();
+/// ```
+/// 
+/// ```
+/// let owned_str = "hello".to_string().to_value();
+/// assert_eq!(owned_str.owned_get::<String>(), "hello".to_string());
+/// ```
+pub unsafe trait FromOwnedValue: for<'a> FromValue<'a> + 'static {
```

Comment:
I wonder if we can make a blanket implementation for this instead of implementing it for every type. Does something like

```rust
unsafe impl<'a, T: FromValue<'a> + 'static> FromOwnedValue for T {}
```

work?

Reply:
That can't work because we don't have a FromValue impl for String. If we manually implement FromOwnedValue for String only, it complains that it's already implemented as part of the blanket implementation :/